nodejs/week2/meal-sharing/src/backend/api/meals-router.js

```diff
@@ -0,0 +1,68 @@
+const { query } = require("express");
+const express = require("express");
+const router = express.Router();
+
+const meals = require("./../data/meals.json");
+
+router.get("/", async (request, response) => {
+  try {
+    const supportedParams = ["maxPrice", "limit", "title", "createdAfter"]; //will be needed for query validation
+    const maxPrice = parseInt(request.query.maxPrice); //converting to int
+    const limit = parseInt(request.query.limit); //converting to int
+    const title = request.query.title;
+    const createdAfter = request.query.createdAfter;
+
+    //Checking if there are any queries
+    if (Object.keys(request.query) != 0) {
```

Comment:
Object.keys never returns 0 so the branch's condition is always true regardless of the query parameters.
Can you fix this?

Reply:
Sure! It should be Object.keys(request.query).length I guess? Lost this "length" part while modifying the code several times.